### PR TITLE
add asia friendly community call

### PR DESCRIPTION
**Pull request by suhuruli**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -69,6 +69,7 @@ One of the easiest ways to contribute is to participate in discussions. There ar
 |---------------------|---|
 | Mailing List | https://groups.google.com/forum/#!forum/oam-dev |
 | Community meeting info | [Bi-weekly (Starting Oct 22, 2019), Tuesdays 10:30AM PST](https://calendar.google.com/calendar?cid=dDk5YThyNGIwOWJyYTJxajNlbWI0a2FvdGtAZ3JvdXAuY2FsZW5kYXIuZ29vZ2xlLmNvbQ)  |
+| APAC Friendly Community meeting | [Bi-weekly (Starting Oct 28, 2019), Mondays 5:00PM PST](https://calendar.google.com/calendar?cid=dDk5YThyNGIwOWJyYTJxajNlbWI0a2FvdGtAZ3JvdXAuY2FsZW5kYXIuZ29vZ2xlLmNvbQ)  |
```
Comment (resouer) [inline README.md]: Emm, the calendar link is same?

Also, I noticed the APAC community meeting link in https://github.com/oam-dev/rudr#contributing seems not working ("not found")?

Comment (suhuruli) [reply README.md]: The calendar link is the same because the calendar contains all the events. 

- Let me take a look and fix that other link.

Comment (wonderflow) [reply README.md]: This is fine. I think Mondays 4:00PM PST is also OK, if it's possible for matt to join.

**Comment by hongchaodeng** [comment]
@suhuruli

Let's change this to align with https://github.com/oam-dev/rudr/pull/485

The calendar link should be fixed now.

**Comment by suhuruli** [comment]
> @suhuruli
> 
> Let's change this to align with [oam-dev/rudr#485](https://github.com/oam-dev/rudr/pull/485)
> 
> The calendar link should be fixed now.

@hongchaodeng , just did this in the latest PR. I think we can merge this now?

File: README.md [hunk 2]
```diff
@@ -69,7 +69,7 @@ One of the easiest ways to contribute is to participate in discussions. There ar
 |---------------------|---|
 | Mailing List | https://groups.google.com/forum/#!forum/oam-dev |
 | Community meeting info | [Bi-weekly (Starting Oct 22, 2019), Tuesdays 10:30AM PST](https://calendar.google.com/calendar?cid=dDk5YThyNGIwOWJyYTJxajNlbWI0a2FvdGtAZ3JvdXAuY2FsZW5kYXIuZ29vZ2xlLmNvbQ)  |
-| APAC Friendly Community meeting | [Bi-weekly (Starting Oct 28, 2019), Mondays 9:00PM PST](https://calendar.google.com/calendar?cid=dDk5YThyNGIwOWJyYTJxajNlbWI0a2FvdGtAZ3JvdXAuY2FsZW5kYXIuZ29vZ2xlLmNvbQ)  |
+| APAC Friendly Community meeting | [Bi-weekly (Starting Oct 28, 2019), Mondays 1:00PM GMT+8](https://calendar.google.com/calendar?cid=dDk5YThyNGIwOWJyYTJxajNlbWI0a2FvdGtAZ3JvdXAuY2FsZW5kYXIuZ29vZ2xlLmNvbQ)  |
```
Comment (hongchaodeng) [inline README.md]: Can you copy this line:

```
[Bi-weekly APAC (Starting Dec 24, 2019), Tuesdays 1:00PM GMT+8](https://calendar.google.com/event?action=TEMPLATE&tmeid=MzJnbHR2b3R1bHYxMG0wc2YybDJjZmhuc2pfMjAxOTEyMjRUMDUwMDAwWiBmZW5namluZ2NoYW9AbQ&tmsrc=fengjingchao%40gmail.com&scp=ALL)
```

There are nuance.

**Comment by hongchaodeng** [comment]
LGTM after nit

**Comment by wonderflow** [comment]
Yes，please change to ` Tuesdays 1:00PM GMT+8`

**Comment by hongchaodeng** [comment]
@suhuruli 
Can you make the quick change?

**Comment by suhuruli** [comment]
@hongchaodeng and @wonderflow , please merge when ok

**Pull request merged**